cmd/heplify-server/heplify-server.go

```diff
@@ -44,6 +45,22 @@ func init() {
 	} else {
 		fmt.Println("Could not find toml config file, use flag defaults.", err)
 	}
+	
+	
+	if len(config.Setting.AlegIDs) > 0 {
```

Comment:
Could you put this block into a function or so you can reuse it in the test.

Reply:
done